### ISSUE TITLE
Button placement and images seems to be all jacked up

### ISSUE THREAD

**Issue by thinkpad4** [devunt/make-gis-great-again#71]
@devunt  Google changed something again

- Are you using browser extension or userscript?
Browser extension
- Is "Make GIS great again" working on other pages? 
- Is error disappeared when you changed extension/userscript to userscript/extension?
- Link to page where error occurs
All Google images when you click on them
- Screenshot of page where error occurs
https://i.imgur.com/f5wA2uI.jpg
- Screenshot of Developer tools' "Console" (Firefox — `Control+Shift+K`, Chrome — `Ctrl+Shift+J`)

**Comment by DonaldLuck** [comment]
And, now, completely changed.
![screenshot_20180403173046](https://user-images.githubusercontent.com/36758163/38277122-4d225334-3765-11e8-8177-f24633760cce.jpg)
![screenshot_20180403173127](https://user-images.githubusercontent.com/36758163/38277123-4d7b641a-3765-11e8-97f0-bec3f00dc9ca.jpg)

**Comment by Tampa** [comment]
Yes, on .de .com and .fr, seemingly all version of image search now look like that or similar. Apart from the design not working on some browsers and looking completely broken on mobile it also hides the useful buttons even further.

Likely that a complete rewrite of this plugin is needed to support this new design. Wouldn't help to then also fix the issues this new design generates for mobile and some browsers.

![firefox_2018-04-09_19-56-28](https://user-images.githubusercontent.com/1579370/38513992-337c3006-3c30-11e8-85c5-53127bdde280.png)

**Comment by perdolka** [comment]
Weird. 
Google switched back to old design, so script works again.
